reference/configuration/framework.rst

```diff
@@ -552,145 +1151,74 @@ to implement the :class:`Symfony\\Component\\Validator\\Mapping\\Cache\\CacheInt
 enable_annotations
 ..................
 
-**type**: ``Boolean`` **default**: ``false``
+**type**: ``boolean`` **default**: ``false``
 
 If this option is enabled, validation constraints can be defined using annotations.
 
 translation_domain
 ..................
 
-**type**: ``string`` **default**: ``validators``
+**type**: ``string`` **default**: ``'validators'``
 
 The translation domain that is used when translating validation constraint
 error messages.
 
-Full default Configuration
---------------------------
```

Comment:
funny you say this, as you were the one against it a couple of years back :) I've not yet removed it (will do in another PR), so we can discuss it again.